book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] @iamchristianani great work in your class however in order to achieve many to many relationship between book and person your add rental method needs a book and date parameters, so that the rental to be created should have three parameters the date, the person himself and the book being rented, kindly modify the method.